app/views/content/life-as-a-teacher/age-groups-and-specialisms/primary/_article.html.erb

```diff
@@ -0,0 +1,91 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach primary</h2>
+    <div class="inset">
+      <p>Becoming a primary school teacher is an incredibly rewarding career. You'll work with young pupils who are eager to learn and see them develop from learning foundational skills like holding a pencil to becoming independent learners.</p>
+      <p>Teaching primary is a great fit for anyone who enjoys a broad curriculum, as you will teach everything from maths to physical education. You'll create a nurturing enviroment in which pupils feel safe, grow in confidence and have the skills neccessary for secondary school.</p>
+      <!-- Why teach: benefits -->
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_primary_jodie' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+   <p>Pupils start primary school in England when they are 4 and leave when they are 11 years old. They will usually be taught by one teacher who will be responsible for a class for the year.</p>
```

Comment:
```suggestion
   <p>Pupils start primary school in England when they are 4 and leave when they are 11 years old. They will usually be taught by 1 teacher who will be responsible for a class for the year.</p>
```

Reply:
We normally write numbers as digits, rather than words, but this might look a bit odd?